prereise/gather/demanddata/transportation_electrification/data_helper.py

```diff
@@ -89,3 +90,56 @@ def update_if_weekend(data: pd.DataFrame):
     data.loc[data["Day of Week"].isin(range(2, 7)), "If Weekend"] = 2
 
     return data
+
+
+def get_total_daily_vmt(
+    data: pd.DataFrame,
+    comm_type: int,
+    locationstrategy: List[int],
+    input_day: List[int],
+    data_day: np.array,
+):
+    """Load data and use the parameters to calculate total_daily_vmt.
+
+    :param pandas.DataFrame data: the data returned from :func:`load_data`.
+    :param int comm_type: the type of Commute
+    :param List[int] locationstrategy: strategy for each location
+    :param List[int] input_day: day of the week for each day in the year derived from
+        first_func
```

Comment:
Can we use `list` for the type in the docstring to follow our standard. Also what is `first_func`?

Reply:
done